src/client/Network.js

```diff
@@ -112,7 +108,7 @@ export default class Network extends ManagedNetwork {
      */
     get networkName() {
         return this._ledgerId != null
```

Comment:
If you change `this._ledgerId` to be of type `NetworkName` you wouldn't need to convert it in the conditional.